pages/home.page.ts

```diff
@@ -13,6 +13,11 @@ export class HomePage {
     this.sortDropdown = this.page.getByTestId('sort');
   }
 
+  async openHomePage() {
+    await this.page.waitForTimeout(3000);
```

Comment:
Please find another way to wait for an element or page load. Sleep is a bad practice